src/DynamoCoreWpf/UI/GuidedTour/StepUIAutomation.cs

```diff
@@ -97,7 +97,7 @@ public enum UIControlType { MENUITEM, BUTTON, FUNCTION, JSFUNCTION };
         /// <summary>
         /// This property is the position of the node that will be placed
         /// </summary>
-        internal Point2D NodePosition { get; set; }
+        public Point2D NodePosition { get; set; }
```

Comment:
@filipeotero and I were discussion this, we made it internal on purpose, any reason it has to be public?

Reply:
@QilongTang yesterday I commented @filipeotero about it, it was not deserializing because is internal, then the position was always 0,0 after making it public was deserializing the values from the json file correctly, please let me know if it sounds good to you.